controllers/calendar/helpers/RestDefinitions.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace humhub\modules\smartVillage\controllers\calendar\helpers;
+
+use humhub\modules\calendar\models\CalendarEntry;
+use humhub\modules\calendar\models\CalendarEntryParticipant;
+use humhub\modules\rest\definitions\ContentDefinitions;
+use humhub\modules\rest\definitions\UserDefinitions;
+
+class RestDefinitions
+{
+    public static function getCalendarEntry(CalendarEntry $entry)
+    {
+        return [
+            'id' => $entry->id,
+            'title' => $entry->title,
+            'description' => $entry->description,
+            'start_datetime' => $entry->start_datetime,
+            'end_datetime' => $entry->end_datetime,
+            'all_day' => (int)$entry->all_day,
+            'participation_mode' => (int)$entry->participation_mode,
+            'color' => $entry->color,
+            'allow_decline' => (int)$entry->allow_decline,
+            'allow_maybe' => (int)$entry->allow_maybe,
+            'time_zone' => $entry->time_zone,
+            'location' => $entry->location ?? "",
+            'participant_info' => $entry->participant_info,
+            'closed' => (int)$entry->closed,
+            'max_participants' => (int)$entry->max_participants,
+            'content' => ContentDefinitions::getContent($entry->content),
+            'participants' => static::getParticipantUsers($entry->getParticipantEntries()->with('user')->all())
+        ];
```

Comment:
please add the original code or a link to it in such cases, so that we can make a review comparing the code

Reply:
it might be this, right? https://github.com/humhub/calendar/blob/0fc823cce4a0a32e41133f0712842e9a404542ce/helpers/RestDefinitions.php

Reply:
Okay, @donni106